code/modules/events/spacevine.dm

```diff
@@ -9,7 +9,7 @@
 

```

Comment:
How it works:
On this line you make a var/obj/effect/spacevine/SV = new(null)

Then in the loop you do if(F.Enter(SV)) instead of all other checks